### ISSUE TITLE
Doesnt work with printed marker

### ISSUE THREAD

**Issue by Zumbalamambo** [EnoxSoftware/MarkerBasedARExample#2]
why is that it is not working with the printed marker?

**Comment by EnoxSoftware** [comment]
Have you tested with this image?
https://github.com/EnoxSoftware/MarkerBasedARExample/blob/master/Assets/MarkerBasedARExample/Resources/marker.png

**Comment by Zumbalamambo** [comment]
@EnoxSoftware Yes, I printed the exact marker but it just doesnt detect it! :*(

**Comment by EnoxSoftware** [comment]
Could you tell me about your test environment?
OpenCVForUnity version :
MarkerBasedARExample version : 
Unity version :
Build Platform : 
Is the camera front camera or rear camera? : 

Also,
Is the marker design set as shown in the image below?
<img width="546" alt="marker" src="https://user-images.githubusercontent.com/7920392/64517579-977b3780-d32b-11e9-9870-339ca09fc8f6.PNG">
